gst/tensor_decoder/tensordec.c

```diff
@@ -372,9 +372,11 @@ _tensordec_process_plugin_options (GstTensorDec * self, int opnum)
  */
 #define PROP_MODE_OPTION(opnum) \
     case PROP_MODE_OPTION ## opnum: \
-      self->option[opnum - 1] = g_value_dup_string (value); \
+      self->option[(opnum) - 1] = g_value_dup_string (value); \
       if (self->mode == DECODE_MODE_PLUGIN) { \
-        g_assert (_tensordec_process_plugin_options (self, opnum - 1) == TRUE); \
+        if (_tensordec_process_plugin_options (self, (opnum) - 1) != TRUE) \
+          GST_ERROR ("Configuring option for tensor-decoder failed (option %d = %s)", \
```

Comment:
how about changing to GST_ERROR_OBJECT (self, ..)

Reply:
Done!